source/tutorial/5_setup.rst

```diff
@@ -57,12 +57,14 @@ Module Description
 Now that we have a clear idea of what our apps will do, we can begin to build it. Because this example is used in training and is built by many users, the ``{uid}`` is added to make sure that your module has a unique
 name. However, you would not usually put your own username in the module name, and instead name it something like ``ContigFilter``.
 
+If not performed already, for convenience define these variables: ``your_kbase_user_name=jane.smith`` and ``uid=jsmith``. Because we are working to implement a new module, re-define the module name variable: ``module_name=ContigFilter``.
```

Comment:
If you want the current bash username, you can use the standard `$USER` instead of defining a new variable. Maybe we should use the kbase username in the module name because we know that will be a unique name. Bash username may not be... (eg. my bash username is just "j")

Reply:
This was somewhat confusing on my end. 'uid' is never actually defined (I believe), so I also first assumed that it meant 'user ID'. The more I explored, the more it seems like it means 'unique ID'. What if this was a combination of $USER and date - for example, ${USER}_$(date +%F) . This would create a unique ID for each user running the tutorial, but it does assume that users clear certain tutorial checkpoints within a day.

I defer to you, the expert.

Reply:
I would say do `username=your_kbase_username` and use `${username}` everywhere in place of `uid`.

Reply:
I like it. Consider it done.